lib/authentication.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+const MAX_PASS_LENGTH = 128;
+const MIN_PASS_LENGTH = 10;
+const MIN_PHRASE_LENGTH = 20;
+const MIN_OPTIONAL_TESTS = 3;
+
+const MIN_LOGIN_LENGTH = 3;
+const MAX_LOGIN_LENGTH = 30;
```

Comment:
If user will use email for login we need at least 64 chars (gmail have 255 max email length)